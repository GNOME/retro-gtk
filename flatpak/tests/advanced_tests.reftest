[Retro Reftest]
Path=/advanced_tests
Core=/app/lib/libretro/advanced_tests_libretro.so

[Frame 0]
State=Refresh
Video=advanced_tests.png
