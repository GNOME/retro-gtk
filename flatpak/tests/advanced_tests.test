[Retro Reftest]
Path=/advanced_tests
Core=/app/lib/libretro/advanced_tests_libretro.so

[Frame 0]
Tests=Run;Video;StateRefresh;
Video=advanced_tests.png
