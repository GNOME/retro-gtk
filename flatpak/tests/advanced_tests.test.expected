/advanced_tests/Boot: OK
/advanced_tests/0/Run: OK
/advanced_tests/0/Video: OK
/advanced_tests/0/StateRefresh: OK
